### PR TITLE
feat(all): update versions to 3.2

### PR DESCRIPTION
**Pull request by huwshimi**
## Done

- Update versions to 3.2.
- Update release instructions.

## QA

N/A

## Fixes

Fixes: canonical-web-and-design/app-tribe/issues/924.

### CODE REVIEW

File: .github/workflows/accessibility.yml [hunk 1]
```diff
@@ -1,5 +1,11 @@
 name: accessibility
-on: [push]
+on:
+  push:
+    branches:
+      - 3.2
+  pull_request:
+    branches:
+      - 3.2
```
Comment (petermakowski) [inline .github/workflows/accessibility.yml:8]:
```suggestion
```

Comment (petermakowski) [reply .github/workflows/accessibility.yml:8]: This should be probably the same as the main cypress workflow. Running these on a PR won't detect any new issues anyway. As you know we're running tests against a snap build, not latest maas-ui source code.

**Review by petermakowski — APPROVED** [review]
LGTM, just a minor change I requested below. Appreciate you're keeping the docs up to date. 👍

File: .github/workflows/accessibility.yml (hunk 1 above)
Comment (huwshimi) [reply .github/workflows/accessibility.yml:8]: Thanks @petermakowski!

**Pull request merged**